platform/working-with-vsp/orientation/request-access-to-tools.md

```diff
@@ -1,91 +1,94 @@
 
-In addition to internal tools our infrastructure is hosted in and makes use of services in AWS GovCloud. This account is shared amongst many different teams; changes made to resources in this account will affect many services. You will need access to AWS for troubleshooting, log file review, and to apply changes to the running instances and services. 
+2. Additional details and orientation on ZenHub can be found [in this document](./zenhub.md).
 
-For either SOCKS5 proxy and/or AWS Console access please create a "Environment Access Request" issue in Github. All SOCKS5 proxy access requests are reviewed by VSP Operations. 
+## Additional access for developers
 
-In order to be granted AWS Console access, your PIV background (eQIP adjudication) check needs to be underway(for VSP Operations folks, this process is expedited). Please mark yes in the AWS Console access and provide a short description of why you need access. All AWS Console access requests will be reviewed by DEPO. **If you do not need access to AWS please mark "No" in the Grant AWS Console Access section**. Please ensure  all VA onboarding paperwork is submitted to your company/COR.
+Several internal tools for development are only accessible within the VA network. Access to developers outside the VA network is made available via a SOCKS proxy. For proxy access, we require the use of an SSH key pair for security.
 
-We will be walking through how to request access in the steps outlined below.
+Follow the below steps to set up your SSH key pair, request access, and set up the SOCKS proxy:
 
-#### 1. Create new SSH keys for [Windows](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#creating-an-ssh-keypair-in-windows) or [Mac/Linux](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#create-ssh-public-key).
+1. Create new SSH keys.
+    * [Windows instructions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#creating-an-ssh-keypair-in-windows)
+    * [Mac/Linux instructions](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#create-ssh-public-key).
 
-#### <a id="authorizekeys"></a>2. Request that your SSH keys be authorized so that you can use the developer tools such as Jenkins, Grafana and Sentry.
-* Open an [Environment Access Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=external-request%2C+operations&template=Environment-Access-Request-Template.md&title=Access+for+%5Bindividual%5D) issue in the [va.gov-team](https://github.com/department-of-veterans-affairs/va.gov-team) repo.
+1. <a id="authorizekeys"></a> Open an [Environment Access Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose) issue.
   * Follow the template instructions.
```

Comment:
Consider removing the explanation of how to follow the template instructions that are fairly self-evident and just focusing on the AWS stuff that seems like it could be a point of confusion.